File: Pratica 4 - Estrutura de Dados SImples/cpp_logic/exercicio2.cpp
#include <iostream>
using namespace std;

int main() {
    double A[20], B[20], C[20];

    cout << "Digite os valores de A:";
    for (int i = 0; i < 20; i++) {
        cin >> A[i];
    }

    cout << "Digite os valores de B:";
    for (int i = 0; i < 20; i++) {
        cin >> B[i];
    }

    for (int i = 0; i < 20; i++) {
        C[i] = A[i] - B[i];
    }

    cout << "Matriz C: ";
    for (int i = 0; i < 20; i++) {
        cout << C[i] << " ";
    }

    return 0;
}

File: Pratica 4 - Estrutura de Dados SImples/python_logic/exercicio2.py
matriz_A = []
matriz_B = []
matriz_C = []

print("Digite 20 valores para A:")
for i in range(20):
    valor = float(input(f"A[{i}]: "))
    matriz_A.append(valor)

print("\nDigite 20 valores para B:")
for i in range(20):
    valor = float(input(f"B[{i}]: "))
    matriz_B.append(valor)

for i in range(20):
    matriz_C.append(matriz_A[i] - matriz_B[i])

print("Matriz C (A - B):")
print(matriz_C)